### PR TITLE
Add tags for network connectivity tests

### PR DESCRIPTION
**Pull request by msampathkumar**
Add region tags for existing code samples.

If this PR is for Terraform, I acknowledge that I have:

- [ ] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-downstream-tools), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/third_party/terraform/tests) (for handwritten resources or update tests).
- [ ] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [ ] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

File: mmv1/templates/terraform/examples/network_management_connectivity_test_addresses.tf.erb [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# [START networkconnectivity_test_addresses]
```
Comment (betsy-lichtenberg) [inline mmv1/templates/terraform/examples/network_management_connectivity_test_addresses.tf.erb:1]: The region tag prefix should be networkconnectivitycenter, as shown here:

https://devrel.corp.google.com/admin/products/415?t=snippets

Comment (c2thorn) [reply mmv1/templates/terraform/examples/network_management_connectivity_test_addresses.tf.erb:1]: @betsy-lichtenberg it looks like the product is actually Network Management, not Network Connectivity Center. So I believe it should be `networkmanagement`

**Review by c2thorn — CHANGES_REQUESTED** [review]
please review the region tag

File: mmv1/templates/terraform/examples/network_management_connectivity_test_addresses.tf.erb (hunk 1 above)
Comment (msampathkumar) [reply mmv1/templates/terraform/examples/network_management_connectivity_test_addresses.tf.erb]: I have update the tag now. Thank you @betsy-lichtenberg  @c2thorn

**Review by c2thorn — APPROVED** [review]

**Review by c2thorn — CHANGES_REQUESTED** [review]
@msampathkumar your fork is a couple of months behind [GoogleCloudPlatform:master](https://github.com/GoogleCloudPlatform/magic-modules) which is old enough for this merge not to work. You should sync your fork by pulling the latest upstream changes.

Then you'll have to rebase your fork's branch `msampathkumar:terraform-add-network-connectivity-tests` onto the latest commits from `GoogleCloudPlatform:master` (or start a new branch from `msampathkumar:master`)

**Comment by msampathkumar** [comment]
Hi @c2thorn, Apologies for the trouble. I did not notice this earlier.

Now, I have updated my fork and branch. Please check. Otherwise I will raise a fresh PR. Thank you.

**Review by c2thorn — APPROVED** [review]

**Pull request merged**